### DISCUSSION TITLE
Closest Point

### DISCUSSION THREAD

**Discussion by feersantana5** [isl-org/Open3D#3707] [Q&A] [answered]
Is there any method to find the closest point in the cloud (in axis Z)?

**Comment by ssheorey** [comment, marked as answer]
There are multiple ways to do that (e.g.: nearest neighbors, or just scanning through all the points). The best method will depend on your application. Here is one possible method:
http://www.open3d.org/docs/latest/tutorial/geometry/pointcloud.html#Point-cloud-distance
